test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -196,52 +186,6 @@ public int waitFor() throws InterruptedException
         return this;
     }
 
-    private void watchIO()
-    {
-        OutputStream in = process.getOutputStream();
-        InputStream err = process.getErrorStream();
-        InputStream out = process.getInputStream();
-        while (true)
-        {
-            boolean errHandled;
-            boolean outHandled;
-            try
-            {
-                if (stdin != null)
-                {
-                    IOUtils.copy(stdin, in);
-                    if (stdinAutoClose)
```

Comment:
In trying to add stdin support back bound that this logic doesn't make sense.  `IOUtils.copy` blocks waiting for the stream to close, so doesn't make sense to close the closed stream.